app/assets/stylesheets/layout_components/comparison-radios.scss

```diff
@@ -36,6 +36,43 @@
     text-align: center;
   }
 
+  &__radio-btn--stacked{
+    .comparison-radios__input + label{
+      @include breakpoint(small only){
+        overflow: hidden;
+      }
+
+      @include breakpoint(small){
+        padding: 0 0 46px 0;
```

Comment:
does this need to be 46px? or could we do $base-padding*9, which would give us 45px. We just try to use our variables instead of "magic numbers" whenever possible.